gap/oper.gd

```diff
@@ -97,3 +97,7 @@ DeclareOperation("PartialOrderDigraphJoinOfVertices",
 DeclareOperation("PartialOrderDigraphMeetOfVertices",
 [IsDigraph, IsPosInt, IsPosInt]);
 DeclareOperation("DigraphClosure", [IsDigraph, IsPosInt]);
+
+DeclareOperation("IsMatching", [IsDigraph, IsList]);
```

Comment:
These three things should be declared for `IsHomogeneousList` rather than `IsList`.